### PR TITLE
2.0 update

### PR DESCRIPTION
**Pull request by mezotv**
## Description

**Link to the issue (if applicable)**:
Provide a link to the related issue (e.g., #[issue_number]).

**Description**:
Explain the purpose and context of this PR.

## Changes Made

List the changes you made in this PR.

## Testing

Describe the testing process you followed to ensure that the changes in this PR are effective and do not introduce new issues. Include information about the test cases you ran or any manual testing steps.

## Screenshots (if applicable)

Add screenshots or GIFs to showcase the changes visually.

## Additional Notes

Include any additional information or context that reviewers should be aware of.

### CODE REVIEW

File: src/cluster.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
 import "dotenv/config";
 import { ClusterManager } from "discord-hybrid-sharding";
+import net from "node:net";
 
 const manager = new ClusterManager(`${__dirname}/index.js`, {
-  shardsPerClusters: 10,
```
Comment (mezotv) [inline src/cluster.ts]: This needs to be changed back to 10.

**Comment by Sans3108** [comment]
I only now realized I was supposed to name each function for the settings sub-commands differently but I forgot, it doesn't matter anyway because of how they're imported but it's worth noting
[4660c7a](https://github.com/Would-You-Bot/client/pull/432/commits/4660c7ad64a26c47e97446b5702d4dbba219bf14)

**Comment by mezotv** [comment]
I really need to fix the merge conflicts damn.

**Comment by mezotv** [comment]
> I really need to fix the merge conflicts damn.

I did!

**Pull request merged**